### DISCUSSION TITLE
CRD upgrades when deploying via the helm chart

### DISCUSSION THREAD

**Discussion by EugenMayer** [longhorn/longhorn#10030] [General]
Checked the docs and could not find anything releated. The CRDS are changing, and AFAICs there are already issues running 1.7.2 with outdated CRDs (backing image) https://github.com/longhorn/longhorn/blob/master/chart/templates/crds.yaml

What are the expected upgrade procedures right now, esp. in regard of CRDS?

Thankks

**Comment by derekbit** [comment]
> Checked the docs and could not find anything releated. The CRDS are changing, and AFAICs there are already issues running 1.7.2 with outdated CRDs (backing image) [master/chart/templates/crds.yaml](https://github.com/longhorn/longhorn/blob/master/chart/templates/crds.yaml)


Can you provide more information? 
[master/chart/templates/crds.yaml](https://github.com/longhorn/longhorn/blob/master/chart/templates/crds.yaml) is for master (development) branch rather than v1.7.2.
